src/lib/registration/ssl_certificate.rb

```diff
@@ -175,12 +191,15 @@ def import_to_system
     # to installation system is slightly different to the one followed
     # to import certificates to a installed system.
     #
+    # @param target_path [String] where the imported certificate will be saved,
+    #   the pathe should contain the INSTSYS_CERT_DIR prefix otherwise it might
+    #   not work correctly.
     # @return [Boolean] true if import was successful; false otherwise.
     #
     # @see update_instsys_ca
-    def import_to_instsys
+    def import_to_instsys(target_path = self.class.default_certificate_path)
```

Comment:
to be honest I do not like much this new param as it is confusing when seeing it in use. consider `certificate.import_to_instsys(path)` this for me looks like it imports from path and not to path. So what about named param `to`? like `def import_to_instsys(to: self.class.default_certificate_path)` so usage will looks like `certificate.import_to_instsys(to: path)` ?

Reply:
Um, the usual usage looks like
```ruby
cert = SslCertificate.load_file(cert_file)
cert.import_to_instsys(target_path)
```

Where it is IMHO obvious what is loaded from and saved to.

I do not like adding a parameter name if there is only one parameter. If the name is confusing we could change the method name to `save_to_instsys(path)` or something like that.

Reply:
well, method `save_to_instsys`  makes good sense for me.

Reply:
but just NP, so feel free to merge.

Reply:
btw, here I do not get whole import name, when you actually exporting certificate and do not import anything.